app/services/contract_event_payment_calculator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "initialize_with_config"
```

Comment:
can't this be autoloaded with zeitwerk?

Reply:
We seem to be using :classic as an autoloader. 
`    
  config.autoloader = :classic
`